docs/api/class-reference.md

```diff
@@ -91,6 +95,19 @@ This object represents a generic Creative. It's used as a parent object for more
 - `mediaType: String|null` Type of media file (3D / 360 / etc). Optional. Default value = 2D
 - `fileURL: String|null`
 
+## InteractiveCreativeFile<a name="interactivecreativefile"></a>
+
+- `type: String` MIME type for the file provided
+- `apiFramework: String` Identifies the API needed to execute the resource file, if applicable
+- `variableDuration: Boolean`  Identifies whether the ad always drops when the duration is reached, or if it can potentially extend the duration by pausing the underlying video or delaying the adStopped call after adVideoComplete
+- `fileURL: String|null`
```

Comment:
it would be great to specify which values could be optional (`null`)

Reply:
Done